inc/ocf_logger.h

```diff
@@ -12,7 +12,17 @@
  */
 
 #include <ocf/ocf_types.h>
-#include <stdarg.h>
+
+#ifndef __KERNEL__
+	#include <stdarg.h>
+#else
+	#include <linux/version.h>
+	#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 15, 0)
+		#include <linux/stdarg.h>
+	#else
+		#include <stdarg.h>
+	#endif
+#endif
```

Comment:
We want to keep OCF entirely platform agnostic. If there is any dependency (like this one) which can differ between different platforms, it should be rather handler through ocf_env.

Reply:
Did you mean something like this?

Reply:
Yes. In posix env we can also drop `#ifndef __KERNEL__` as this is always userspace env. That kernel part should be implemented in kernel env defined in [Open CAS Linux ](https://github.com/Open-CAS/open-cas-linux) repository.

Reply:
Ok. I just replaced stdarg.h with ocf_env.h in ocf_logger and I'll submit a new patch for open-cas's env file.